README.md

```diff
@@ -57,9 +75,31 @@ a foundation for building new systems. Here are some example use cases:
 - _Easy to Embed_: Allowing extension at almost any point in its design, DataFusion can be tailored for your specific use case
 - _High Quality_: Extensively tested, both by itself and with the rest of the Arrow ecosystem, DataFusion can be used as the foundation for production systems.
 
+## Comparisons with other projects
```

Comment:
How about adding databend?

Reply:
It is a good idea -- thank you @xudong963 

I have not been asked personally about the difference between Databend and DataFusion and I am not very knowledgable about Databend.

Would it be correct to say something like "Databend is complete warehouse system, also written in rust, targeting end users rather than other developers of database systems"?

Reply:
Yes, it's a warehouse system, we are also planning to expand it to lakehouse in the next phase: part of lakehouse in https://github.com/datafuselabs/databend/issues/9448.

Btw, databend is available for application developers, some of whom build saas applications based on databend.

I think we can write it as "Databend is a complete and powerful cloud data warehouse with excellent performance, also written in rust, targeting end users rather than other developers of database systems".  -- Thanks @alamb

Reply:
In general (not just for Databend), I think it is a good idea to give explanations that are concise and focus on functionality/context — while sounding less marketing-like. In this case, I suggest something like this:

Databend is a complete, high-performance warehouse system, inspired by ClickHouse and Snowflake, targeting end-users rather than developers of other database systems. It is also written in Rust and utilizes Apache Arrow.

This should give enough context for anyone interested to dig deeper in Databend’s website (which is great BTW, I love how progress on benchmarks are shown over time).

Reply:
Thanks @ozankabak

Reply:
I added a note in 020fe58a0

Reply:
@ozankabak  @alamb  There is a general benchmark in https://github.com/ClickHouse/ClickBench,  databend will update the results in recent days. 

I think datafusion could update the results of the latest version in`c6a.4xlarge, 500gb gp2`.

Reply:
Thank you @sundy-li  -- @waitingkuo  I wonder if you have time to update the benchmark results?

Reply:
Updating benchmark results is a good idea. I expect us to get progressively more attention from the community at large, and it would be good to accurately (re)present Datafusion's capabilities/performance.